index.bs

```diff
@@ -5939,7 +5943,8 @@ refer to a type whose values are the set of strings (sequences of
 
 Like {{DOMString}}, there is no way to represent a constant [=enumeration=]
 value in IDL, although enumeration-typed [=dictionary member=]
-[=dictionary member/default values=] can be specified using a
+and [=optional argument|operation optional argument=] default values
```

Comment:
This reads confusingly to me; it's not clear the "default values" distributes over the two preceding constructs. I suggest just distributing it manually, i.e. "enumeration-typed dictionary member default values and operation optional argument default values". That way you can also link to the correct "default values" in both cases.